app/views/users/new.html.erb

```diff
@@ -0,0 +1,49 @@
+<header>
+  <style>
+    .container {
+      margin: auto;
+      width: 50%;
+      border: 3px solid green;
+      padding: 10px;
+      position: relative;
+      text-align: center;
+      justify-content: center;
+    }
+
+  </style>
+
+  <div class="logo">
+    <!-- Logo/App Name -->
+    <%= render partial: '/layouts/partials/app_logo_banner' %>
+    <!-- Logo/App Name -->
+
+    <!-- Navbar -->
+    <%= render partial: '/layouts/partials/nav_bar', locals: {links: {uri: '/', link_text: 'Home'}} %>
+    <!-- Navbar -->
+  </div>
+
+
+</header>
+
+<body>
+  <div class="container">
+    <div class="form-container">
+      <%= form_with url: '/users/create', method: :post, local: true do |f|  %>
+
+      <%= f.text_field :name, required: :true, placeholder: 'Name' %><br><br>
+
+      <%= f.email_field :email, required: true, placeholder: 'email@domain.com' %><br><br>
+
+      <%= f.password_field :password, required: true, placeholder: 'password' %><br><br>
+
+      <%= f.password_field :password_confirmation, required: true, placeholder: 'confirm password' %><br><br>
+
+      <%= f.submit 'Submit' %><br><br>
+      <% end %>
+    </div>
+
+    <p text-align="center">--- OR ---</p><br>
+
+    <%= button_to 'Sign in with Google', '/auth/google_oauth2/', method: :get, params: {client_id: "#{ENV['GOOGLE_CLIENT_ID']}" }  %>
```

Comment:
Is signing in with Google the same as registering with Google?
Will there be login functionality without Google since there is an option to register using a password and password-c?

Reply:
Yes, I agree. There is a Google OAuth button on both the login and register page, but we should definitely change the button text.